crates/owned/src/lib.rs

```diff
@@ -101,7 +101,7 @@ impl<'a> Auth<'a> {
         self.0.save(deps.storage, &state)?;
 
         Ok(Response::default().add_event(
-            Event::new("ownership_transfer_requested")
+            Event::new("ownership_transfer_accepted")
```

Comment:
the other event name actually made sense to me, only this one seems off (looks like a copy paste error)